sync_diff_inspector/config.toml

```diff
@@ -24,6 +24,9 @@ use-checksum = true
 # set true if just want compare data by checksum, will skip select data when checksum is not equal. 
 only-use-checksum = false
 
+# set true will continue check from the latest checkpoint
+use-checkpoint = true
```

Comment:
default `true` or `false`, which is better in this sample config.

Reply:
for `use-checkpoint`, I think default value is true is better